src/Be.Vlaanderen.Basisregisters.GrAr.Extracts/ProjectedCoordinateSystem.cs

```diff
@@ -0,0 +1,21 @@
+namespace Be.Vlaanderen.Basisregisters.GrAr.Extracts
+{
+    using System;
+    using System.IO;
+    using System.Text;
+
+    public class ProjectedCoordinateSystem
+    {
+        private readonly string _projectedCoordinateSystemDefinition;
+        
+        public static ProjectedCoordinateSystem Belge_Lambert_1972 => new ProjectedCoordinateSystem("PROJCS[\"Belge_Lambert_1972\",GEOGCS[\"GCS_Belge_1972\",DATUM[\"D_Belge_1972\",SPHEROID[\"International_1924\",6378388.0,297.0]],PRIMEM[\"Greenwich\",0.0],UNIT[\"Degree\",0.0174532925199433]],PROJECTION[\"Lambert_Conformal_Conic\"],PARAMETER[\"False_Easting\",150000.01256],PARAMETER[\"False_Northing\",5400088.4378],PARAMETER[\"Central_Meridian\",4.367486666666666],PARAMETER[\"Standard_Parallel_1\",49.8333339],PARAMETER[\"Standard_Parallel_2\",51.16666723333333],PARAMETER[\"Latitude_Of_Origin\",90.0],UNIT[\"Meter\",1.0]]");
```

Comment:
@ridingwolf where is this used?

Reply:
Ah, wait, I think I got it, the registers will use this afterwards?

Reply:
In the registries, where you create the file